ui/webui/test/helpers/progress.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import time
+
+HELPERS_DIR = os.path.dirname(__file__)
+sys.path.append(HELPERS_DIR)
```

Comment:
@KKoukiou I see this is used in cockpit and in our tests, but does that make sense? It feels like we're reinventing `import` here...

Reply:
It makes sense if you 're trying to use functions from other directories, other than the one the file is located in. 
This here does not make sense.

Reply:
Sorry, I should have been more specific. The question was about this kind of practice, not this specific example.

I am aware that if I have code that is invoked from a potentially random path, and it wants to import things that are not in python's library or site-packages or whatever, then it does have to manipulate path. I see that Cockpit uses this to import test stuff, at one place that I am aware of.

However, we have this in every test. So the question was actually, do we really need to do this for some deeper reason, or can we just fill the webui tests dirs with `__init__.py` so we can import without these dances everywhere?

Reply:
Eh, actually, yes. We can do the same as cockpit does with `parent.py`and have a sort of "path initializer" that primes python paths and enables imports without all the ugliness.

That also means this case would be sort-of-not-a-blocker, we'd rewrite all of that anyway.

Reply:
Cockpit imports these paths in all test files as well, see 'import parent'

https://github.com/cockpit-project/cockpit/blob/main/test/verify/check-storage-mdraid#L20

[kkoukiou@fedora cockpit]$ git grep 'import parent' | wc -l
72

Reply:
I think we're agreeing? :-)

Reply:
Yes, we can do the parent.py thing!

Reply:
sorry I did not read your reply https://github.com/rhinstaller/anaconda/pull/4179#discussion_r1085439839 before I submitted mine.